services/src/main/java/org/exoplatform/addons/gamification/rest/RealizationsRest.java

```diff
@@ -175,13 +182,17 @@ public Response getReport(@ApiParam(value = "result fromDate", required = true)
                             @QueryParam("toDate")
                             String toDate) {
     RealizationsFilter filter = new RealizationsFilter();
+    Identity identity = ConversationState.getCurrent().getIdentity();
     Date dateFrom = Utils.parseRFC3339Date(fromDate);
     Date dateTo = Utils.parseRFC3339Date(toDate);
     filter.setFromDate(dateFrom);
     filter.setToDate(dateTo);
 
     try {
-      List<GamificationActionsHistoryDTO> gActionsHistoryList = realizationsService.getAllRealizationsByFilter(filter, 0, 0);
+      List<GamificationActionsHistoryDTO> gActionsHistoryList = realizationsService.getRealizationsByFilter(filter,
+                                                                                                               identity,
+                                                                                                               0,
+                                                                                                               0);
```

Comment:
catch `IllegalAccessException` and return a response with status code 403

Reply:
done !